lit_gpt/args.py

```diff
@@ -61,10 +60,27 @@ class EvalArgs:
 
 
 @dataclass
-class IOArgs:
-    """Inputs and outputs related arguments"""
+class LoraArgs:
```

Comment:
This is already in litgpt.lora.Config, why not expose that instead?

Reply:
Because that config subclasses the full config and would include all model args. But that's not something we would wand to expose as configurable in the script, since we're doing finetuning from a checkpoint where these are fixed.

Reply:
I dont love that we have 2 dataclasses with the same arguments. What if we made them separate in the lora module? Or have it import this new args

We can make this breaking change if we find it to be better.

Reply:
I dropped the change from the PR now, I can maybe do this in a follow up.